packages/forms/src/components/ui/text-input/TextInput.module.css

```diff
@@ -101,6 +102,12 @@
       --current-border-color: var(--swui-textinput-border-color-disabled);
     }
 
+    &&.hideBorder {
```

Comment:
Why 2 &&

Reply:
Umm, for precedence. You taught me this trick :)